### ISSUE TITLE
Time speed breaks when using regular bed (not the hay one)

### ISSUE THREAD

**Issue by NaturalBambooStuff** [anegostudios/VintageStory-Issues#2706]
I live in a 2 block high cave and I have normal bed and sometimes when I sleep on it, message appears "Host not responding or overclocked" and I can't stop sleeping until I press something. Then I wake up with my head stuck in the ceiling, but the speed of time that was sped up due to sleeping stays and when I go outside, I can see sun and moon move with huge speed and days also pass by, which influences my garden and plants dry up.

**Comment by Craluminum2413** [comment]
Could you send `server-main.txt` and other logs from "\AppData\Roaming\VintagestoryData\Logs" folder?

**Comment by NaturalBambooStuff** [comment]
[server-chat.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415115/server-chat.txt)
[server-debug.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415116/server-debug.txt)
[server-event.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415117/server-event.txt)
[server-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415118/server-main.txt)
[server-storyevent.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415119/server-storyevent.txt)
[server-worldgen.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415120/server-worldgen.txt)
[client-audit.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415121/client-audit.txt)
[client-chat.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415122/client-chat.txt)
[client-crash.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415123/client-crash.txt)
[client-debug.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415124/client-debug.txt)
[client-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415125/client-main.txt)
[server-audit.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415126/server-audit.txt)
[server-build.txt](https://github.com/anegostudios/VintageStory-Issues/files/11415127/server-build.txt)


> Could you send `server-main.txt` and other logs from "\AppData\Roaming\VintagestoryData\Logs" folder?

Sure, have a look.

**Comment by Wesmania** [comment]
I believe I have a similar issue. Our server crashed and after I restarted it and logged in, everything was sped up as if someone was sleeping, except permanently. With some experiments I was able to restore normal flow of time with "/time speed -1492". 

I don't have any logs, but I did make a copy of our (10 GiB compressed) save file that has this behaviour. If you give me a public SSH key, I can let you in via sftp to download it.
